src/pages/library/datasetBuilder/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const PAGE_PADDING_HEIGHT = 0;
+export const PAGE_PADDING_WIDTH = 3;
+export const PREPACKAGED_CONCEPT_SETS = [{ name: 'Demographics' }, { name: 'All surveys' }];
```

Comment:
Should this be declared with a type, e.g. 
```suggestion
export const PREPACKAGED_CONCEPT_SETS: ConceptSet[] = [{ name: 'Demographics' }, { name: 'All surveys' }];
```